examples/kind-multinode/templates/k8s/konnectivity-agent-ds.yaml

```diff
@@ -53,7 +78,8 @@ spec:
           "--sync-forever",
           "--probe-interval=5s",
           "--service-account-token-path=/var/run/secrets/tokens/konnectivity-agent-token",
-          "--agent-identifiers=ipv4=${HOST_IP}"
+          "--agent-identifiers=ipv4=${HOST_IP}",
+          "--server-count-lease-selector=apiserver.kubernetes.io/identity=kube-apiserver",
```

Comment:
If the lease controller logic lands before this PR is submitted, update the example to use the KNP leases

Reply:
Sounds good!

Reply:
Done.